src/data/CATEGORY_DATA.ts

```diff
@@ -0,0 +1,94 @@
+const CATEGORY_DATA = [
```

Comment:
보통 `UPPER_SNAKE_CASE`와 같은 형태는 전역 상수로 사용합니다. 암묵적인 약속으로요!
따라서 이런 mock data는 아래처럼 쓰는 것이 좋습니다.
```suggestion
const categoryData = [
```